lib/Graph/PlaceholderBindings.cpp

```diff
@@ -20,42 +20,7 @@
 
 using namespace glow;
 
-bool PlaceholderBindings::compare(const PlaceholderBindings *A,
```

Comment:
I think you'll need to keep this around to get the ExecutorTest to work. You should probably revert the name-based lookup (i.e. `const auto *tensorB = B->get(B->getPlaceholderByName(placeholder->getName()));`) but I think you'll need this function in general.

I'm also fine with moving it into the ExecutorTest to discourage people from using it all over the place.

Reply:
Yeah you're right I've added it back